.github/workflows/ci-tests.yml

```diff
@@ -16,21 +16,34 @@ on:
         required: true
         default: false
         type: boolean
-  merge_group:
-    types: [checks_requested]
```

Comment:
Why was this removed?

Reply:
This was removed as we do not have merge queue for pulsar as it is way less busy than monorepo and running the heavy sdk tests multiple times would also hold up monorepo merges as well.